recipe/meta.yaml

```
@@ -17,12 +17,10 @@ build:
     - pycc = numba.pycc:main
     - numba = numba.numba_entry:main
```

From @stuartarchibald's comment https://github.com/conda-forge/numba-feedstock/pull/44#issuecomment-616444248

```suggestion
    - numba = numba.misc.numba_entry:main
```